infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/environment/KubernetesEnvironmentFactory.java

```diff
@@ -97,16 +98,28 @@ protected KubernetesEnvironment doCreate(
                 + "application/x-yaml, text/yaml, text/x-yaml");
     }
 
-    final KubernetesList list =
-        clientFactory.create().lists().load(new ByteArrayInputStream(content.getBytes())).get();
+    final KubernetesList list;
+    try {
+      list =
+          clientFactory.create().lists().load(new ByteArrayInputStream(content.getBytes())).get();
+    } catch (KubernetesClientException e) {
+      String message = e.getCause() == null ? e.getMessage() : e.getCause().getLocalizedMessage();
```

Comment:
`e.getMessage()` but `e.getCause().getLocalizedMessage()` <- Did you make it on the purpose?

Reply:
No, was an accident trying to find a decent way to avoid the newlines and messy cause. Fixed.